arch/arm/src/sama5/sam_ssc.c

```diff
@@ -931,7 +884,7 @@ static struct sam_buffer_s *ssc_buf_allocate(struct sam_ssc_s *priv)
    * have at least one free buffer container.
    */
 
-  ret = ssc_bufsem_take(priv);
+  ret = nxsem_wait_uninterruptible(&priv->bufsem);
```

Comment:
Lets add `nxsem_set_protocol(&priv->bufsem, SEM_PRIO_NONE);`

Reply:
Let's do in the next patch